packages/ui/src/components/Dialog/Dialog.tsx

```diff
@@ -57,7 +59,7 @@ export const Dialog: React.FC<{
           )}
         >
           <div className="flex justify-between items-center w-full">
-            <Title>{title}</Title>
+            <Title className={titleClassName}>{title}</Title>
```

Comment:
Please remove these changes. This is not how we set dialog titles ([example](https://github.com/litespace-org/litespace/blob/2772922b44f7257bcb7e559e5a9cb9fa9e5ebd4c/packages/ui/src/components/Lessons/ManageLesson/ManageLessonDialog.tsx#L279-L288)).